src/main/java/com/fcgl/madrid/points/model/Trophy.java

```diff
@@ -0,0 +1,131 @@
+package com.fcgl.madrid.points.model;
+
+import javax.persistence.*;
+import java.time.Instant;
+
+@Entity
+@Table(name = "trophies")
+public class Trophy {
```

Comment:
missing a parameter to keep track of how many times a user has to complete a certain action. `action_count` in the database design dochttps://docs.google.com/document/d/1fXGAdi6qCu7uIhB6h7vughDXoPy8z5MNVcaSXsQeS04/edit